test/base62.test.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
```

Comment:
Please undo the name change of this file, or make sure it is similar enough to the old file that Git sees it as a file move.  (Also for handler.test.js)